spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/jms/activemq/ActiveMQConnectionFactoryConfiguration.java

```diff
@@ -68,16 +73,12 @@ ActiveMQConnectionFactory jmsConnectionFactory(ActiveMQProperties properties,
 		static class CachingConnectionFactoryConfiguration {
 
 			@Bean
-			@ConditionalOnProperty(prefix = "spring.jms.cache", name = "enabled", havingValue = "true",
```

Comment:
I don't understand why the condition was removed. What am I missing?

Reply:
@snicoll `CachingConnectionFactoryConfiguration` seems to be already guarded with it. Am I missing something?

Reply:
Indeed, time for me to remind [this tweet](https://twitter.com/relizarov/status/1152535099424104448)  🤦‍♂️